tests/testthat/test_civis_ml_utils.R

```diff
@@ -106,3 +106,41 @@ test_that("get_train_template_id reverts to last id if others not available", {
   expect_equal(id, ans)
 })
 
+test_that("coef.civis_ml returns correct coefficients", {
```

Comment:
For multinomial/multioutput I'd prefer that coef returns a matrix here, one column per dv instead of a vector.  
```
  (Intercept)   (Intercept)   (Intercept)    Unnamed: 0    Unnamed: 0    Unnamed: 0   Sepal.Width   Sepal.Width   Sepal.Width 
   5.02471952   11.42495563 -113.97449115   -0.10176131   -0.06364976    1.12927960   11.54905775   -4.28593365  -34.72930265 
 Petal.Length  Petal.Length  Petal.Length   Petal.Width   Petal.Width   Petal.Width 
 -10.59597673    1.48185815    1.61049283   -5.41703176   -0.29744050   56.52995417 
```

See

```
m <- nnet::multinom(Species ~ Sepal.Length, data = iris)
coef(m)
```
(you'll after to delete the `civis::coef` generic first)